packages/shared/src/utils/contract/contract.tsx

```diff
@@ -29,15 +29,28 @@ export const DELAY_TIME_1S_SYMBOL = 500;
 // generation_interval will be provided via API later to help us distinguish between 1-second and 2-second symbols
 export const symbols_2s = ['R_10', 'R_25', 'R_50', 'R_75', 'R_100'];
 
-export const TURBOS = {
-    LONG: 'turboslong',
-    SHORT: 'turbosshort',
+export const CONTRACT_TYPES = {
+    ACCU: 'ACCU',
+    TURBOS: {
+        LONG: 'TURBOSLONG',
+        SHORT: 'TURBOSSHORT',
+    },
+    VANILLA: {
+        CALL: 'VANILLALONGCALL',
+        PUT: 'VANILLALONGPUT',
+    },
 } as const;
-
-export const VANILLALONG = {
-    CALL: 'vanillalongcall',
-    PUT: 'vanillalongput',
-    FX: 'vanilla_fx',
+export const TRADE_TYPES = {
+    ACCUMULATOR: 'accumulator',
+    TURBOS: {
+        LONG: 'turboslong',
+        SHORT: 'turbosshort',
+    },
+    VANILLA: {
+        CALL: 'vanillalongcall',
+        PUT: 'vanillalongput',
+        FX: 'vanilla_fx',
+    },
 } as const;
```

Comment:
can we consolidate this into a single object

Reply:
I guess it's better for semantic to keep them separate. First object (short strings, caps) is used for fields form BE, 2d - for contract's names hardcoded in FE